build.gradle

```diff
@@ -493,8 +516,16 @@ project('standalone') {
         main = "io.pravega.local.LocalPravegaEmulator"
         classpath = sourceSets.main.runtimeClasspath
         systemProperties System.getProperties()
+        print systemProperties
+
+        def extDirs
+        if (System.getProperty("java.ext.dirs") != null) {
+            extDirs = "-Djava.ext.dirs=" + System.getProperty("java.ext.dirs")
+        }
+
+
         systemProperties 'logback.configurationFile' : new File('config/logback.xml').absolutePath
-        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"
+        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n", extDirs
```

Comment:
here, you can pass the `java.ext.dirs` system property using the `systemProperty` method instead of reconstructing the jvmArgs:
`systemProperty("java.ext.dirs", System.getProperty("java.ext.dirs")`
(similar to what you're doing at line 527)